public/js/test/mochitest/head.js

```diff
@@ -234,6 +236,15 @@ function reload(dbg) {
   return dbg.client.reload();
 }
 
+function navigate(dbg, url, ...sources) {
+  const response = dbg.client.navigate(url);
+  if (sources.length == 0) {
+    return response;
```

Comment:
Do we need this `if` statement? You return the response here, but if you are waiting on sources you'll get a different return value. It'd be nice if this function had a consistent return value, and `waitForSourcesToLoad` will do nothing if you pass it an empty array anyway.